common/common-rest/src/test/java/org/apache/servicecomb/common/rest/codec/param/TestBodyProcessor.java

```diff
@@ -61,11 +61,11 @@
   Buffer outputBodyBuffer;
 
   private void createProcessor(Class<?> type) {
-    processor = new BodyProcessor(TypeFactory.defaultInstance().constructType(type));
+    processor = new BodyProcessor(TypeFactory.defaultInstance().constructType(type), false);
```

Comment:
We must not change a test case if it is required. And here should equivalent to true